pkgs/tools/misc/SP800-90B_EntropyAssessment/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bzip2
+, libdivsufsort
+, jsoncpp
+, openssl
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "SP800-90B_EntropyAssessment";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "usnistgov";
+    repo = "SP800-90B_EntropyAssessment";
+    rev = "v${version}";
+    sha256 = "sha256-KZQ7kC0PbBkjLEQZIqYakQ91OvCxruhdfUwiRHtno3w=";
+  };
+
+  buildInputs = [ bzip2 libdivsufsort jsoncpp openssl mpfr ];
+
+  sourceRoot = "source/cpp";
+
+  makeFlags = lib.optionalString stdenv.hostPlatform.isAarch64 "ARCH=aarch64CROSS_COMPILE=aarch64-linux-gnu-";
```

Comment:
`pkgsCross.aarch64-multiplatform.SP800-90B_EntropyAssessment` fails

```suggestion
  makeFlags = lib.optionals stdenv.hostPlatform.isAarch64 [
    "ARCH=aarch64"
    "CROSS_COMPILE=aarch64-linux-gnu-"
  ];
```

These flags appear to be unnecessary too, The build fails with a issue not related to these
```
libdivsufsort-aarch64-unknown-linux-gnu> aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I../include     -g -O2 -c suftest.c
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/ir0j7zqlw9dc49grmwplppc7gh0s40yf-bash-5.2-p15/bin/bash ../libtool --tag=CC   --mode=link aarch64-unknown-linux-gnu-gcc  -g -O2   -o suftest suftest.o ../lib/libdivsufsort.la
libdivsufsort-aarch64-unknown-linux-gnu> libtool: link: aarch64-unknown-linux-gnu-gcc -g -O2 -o suftest suftest.o  ../lib/.libs/libdivsufsort.a
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: suftest.o: in function `main':
libdivsufsort-aarch64-unknown-linux-gnu> /build/libdivsufsort-2.0.1/examples/suftest.c:128: undefined reference to `rpl_malloc'
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: /build/libdivsufsort-2.0.1/examples/suftest.c:129: undefined reference to `rpl_malloc'
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: ../lib/.libs/libdivsufsort.a(libdivsufsort_la-divsufsort.o): in function `divsufsort':
libdivsufsort-aarch64-unknown-linux-gnu> /build/libdivsufsort-2.0.1/lib/divsufsort.c:343: undefined reference to `rpl_malloc'
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: /build/libdivsufsort-2.0.1/lib/divsufsort.c:344: undefined reference to `rpl_malloc'
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: ../lib/.libs/libdivsufsort.a(libdivsufsort_la-divsufsort.o): in function `divbwt':
libdivsufsort-aarch64-unknown-linux-gnu> /build/libdivsufsort-2.0.1/lib/divsufsort.c:371: undefined reference to `rpl_malloc'
libdivsufsort-aarch64-unknown-linux-gnu> /nix/store/7rm8l96q2l6aa5xqd4b60skcyhzdd7b3-aarch64-unknown-linux-gnu-binutils-2.40/bin/aarch64-unknown-linux-gnu-ld: ../lib/.libs/libdivsufsort.a(libdivsufsort_la-divsufsort.o):/build/libdivsufsort-2.0.1/lib/divsufsort.c:372: more undefined references to `rpl_malloc' follow
libdivsufsort-aarch64-unknown-linux-gnu> collect2: error: ld returned 1 exit status
```

Reply:
Thanks for the comment. Don't know how this line got messed up by my editing while force pushing. I have changed now to what you suggested.